discovery_engine_core/web-api/src/bin/ingestion.rs

```diff
@@ -217,14 +224,20 @@ async fn handle_add_data(
         })
         .partition_result();
 
+    let embeddings_duration = start.elapsed().as_secs();
+    info!("Embeddings calculated in {} sec", embeddings_duration);
```

Comment:
:art: we could also add the number of embeddings next to the time
```suggestion
    info!("{} embeddings calculated in {} sec", documents.len(), embeddings_duration);
```